ctags build fails on Fedora 32 (gcc 10)

Building ctags as outlined in the docs fails under the upcoming Fedora 32 release. Works with Fedora 31. Getting the following build failure:

```
/usr/bin/ld: libctags.a(libctags_a-rpmspec.o):/tmp/tmp.6oTRD8ubub/parsers/autoconf.h:24: multiple definition of `autoconfKind'; libctags.a(libctags_a-autoconf.o):/tmp/tmp.6oTRD8ubub/parsers/autoconf.h:24: first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:1815: ctags] Error 1
make[1]: Leaving directory '/tmp/tmp.6oTRD8ubub'
make: *** [Makefile:1288: all] Error 2
```

To reproduce, run the following in a fedora:32 container:

```
dnf install -y \
    autoconf \
    automake \
    gcc \
    git \
    jansson-devel \
    libseccomp-devel \
    libxml2-devel \
    libyaml-devel \
    make \
    pkgconfig \
    python3-docutils 

# universal ctags
TMP=$(mktemp -d)
cd "$TMP"
git clone http://github.com/universal-ctags/ctags.git .
./autogen.sh
./configure
make
make install
```

--- a/circle.yml
+++ b/circle.yml
@@ -8,10 +8,10 @@ jobs:
   #
   # We assume GNU Make is available.
   #
-   fedora31_gmake:
+   fedora32_gmake:
      working_directory: ~/universal-ctags
      docker:
-       - image: docker.io/fedora:31
+       - image: docker.io/fedora:32
      steps:
        - run:
            name: Install Git
@@ -129,4 +129,4 @@ workflows:
       - fedora30_bmake
       - fedora_distcheck
       - centos_make
-      - fedora31_gmake
+      - fedora32_gmake

--- a/parsers/autoconf.h
+++ b/parsers/autoconf.h
@@ -12,7 +12,7 @@
 
 #include "general.h"
 
-enum {
+typedef enum {
 	AUTOCONF_PACKAGE_KIND,
 	AUTOCONF_TEMPLATE_KIND,
 	AUTOCONF_MACRO_KIND,

